src/services/migration/MigrationService.php

```diff
@@ -0,0 +1,51 @@
+<?php declare(strict_types = 1);
+namespace PharIo\Phive;
+
+class MigrationService {
+    /** @var MigrationFactory */
+    private $factory;
+
+    /**
+     * MigrationService constructor.
+     */
+    public function __construct(MigrationFactory $factory) {
+        $this->factory = $factory;
+    }
+
+    public function ensureFitness(): void {
+        $this->runMigrations(true);
```

Comment:
This is imho bad design: It's unexpected that a test for fitness actually does change state.

We should separate the running of migrations from checking the state.